Lib/fontTools/varLib/mutator.py

```diff
@@ -284,7 +284,7 @@ def instantiateVariableFont(varfont, location, inplace=False, overlap=True):
 		gdef = varfont['GDEF'].table
 		instancer = VarStoreInstancer(gdef.VarStore, fvar.axes, loc)
 
-		merger = MutatorMerger(varfont, loc)
+		merger = MutatorMerger(varfont, instancer)
```

Comment:
note how this `instancer` local variable in the line above this was being thrown away (a new VarStoreInstancer was being created inside MutatorMerger's constructor from the same gdef.VarStore and location).
I changed MutatorMerger to take an already loaded VarStoreInstancer object.
But unlike in mutator module, we have already pre-computed all the deltas at that location, so we pass them (as a dict) to the "instancer" parameter of MutatorMerger. The latter expects such an "instancer" to behave like a mapping or sequence (i.e. with a `__getitem__` method) where the keys are VarIdx integers, and the values the deltas at location.